src/components/SelectionList/Search/ReportListItemHeader.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import Checkbox from '@components/Checkbox';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import {useSearchContext} from '@components/Search/SearchContext';
+import type {ListItem, ReportListItemType} from '@components/SelectionList/types';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {convertToDisplayString} from '@libs/CurrencyUtils';
+import {handleActionButtonPress} from '@userActions/Search';
+import CONST from '@src/CONST';
+import type * as OnyxTypes from '@src/types/onyx';
+import ActionCell from './ActionCell';
+
+type MoneyReportHeaderProps<TItem extends ListItem> = {
```

Comment:
I think this name is a mistake?
```suggestion
type ReportListItemHeaderProps<TItem extends ListItem> = {
```